colossus/src/main/scala/colossus/protocols/http/package.scala

```diff
@@ -72,4 +80,13 @@ package object http {
       worker = worker
     )
 
+    */
+  implicit object ByteStringLikeString extends ByteStringLike[String] {
```

Comment:
I think these should go here because otherwise you have to start importing them just to create responses with ByteStrings.  Either that or we should rework this so no imports are needed to create responses from bytestrings.